packages/api/src/api/auth/discord.ts

```diff
@@ -58,21 +49,19 @@ discord.get('/discord/callback', async (req, res) => {
 
       const { id: discordId } = await userResponse.json();
 
-      const { passport } = req.session as unknown as SessionData;
-      const { user: userGithubId } = passport as PassportData;
-      const user = await req.entityManager.findOne(User, { githubId: userGithubId });
+      const user = await req.entityManager.findOne(User, { githubId: req.user });
```

Comment:
I believe it's going to be under `id` 🤔 

```suggestion
      const user = await req.entityManager.findOne(User, { githubId: req.user?.id });
```

Reply:
I tested it, and the req.user just returns a sting (githubId).  Maybe this is an issue that needs to be changed?

Reply:
Maybe it's in another PR that hasn't been merged yet 🤔 We can merge this then